tests/wpt/web-platform-tests/html/semantics/forms/textfieldselection/selection-not-application-textarea.html

```diff
@@ -1,29 +0,0 @@
-<!DOCTYPE html>
```

Comment:
Why is this test file removed?

Reply:
Also explained in the commit message:

> I removed tests/wpt/web-platform-tests/html/semantics/forms/textfieldselection/selection-not-application-textarea.html
because it doesn't seem to add any extra value - the selection API
always applies to textarea elements, and the API is tested elsewhere.

Reply:
I'd rather not remove this haphazardly without contacting the authors or upstream W3C about this. Note that this is not a servo-specific test - any changes you make under `tests/wpt/web-platform-tests` are shared across all browsers.

Reply:
Fair enough, I'll reinstate it.